### ISSUE TITLE
About analytic formula for the integrals

### ISSUE THREAD

**Issue by fate1997** [ifilot/pyqint#40]
Hi,

Thank you so much for your excellent work! I am new to quantum chemistry, and your codes help me a lot in understanding how to turn theory into practice. 

When diving into your codes, I met some difficulties in understanding the integrals part. Is there any reference that includes the analytic formulas that you referred to?

**Comment by ifilot** [comment]
Thank you for the nice words. I am happy to read that the code is to your liking. If you are new to quantum chemistry, you might also appreciate [the book that I wrote on the subject](https://ifilot.pages.tue.nl/elements-of-electronic-structure-theory/).

To answer your question, there a number of useful sources to mention:
* I used [the paper of Taketa et al.](https://journals.jps.jp/doi/10.1143/JPSJ.21.2313) to build the integral evaluation library
* A very nice derivation of the integrals can be found in [the work of Joshua Goings](https://joshuagoings.com/assets/integrals.pdf)
* I can also recommend the paper of [Reine et al.](https://trygvehelgaker.no/Publications/WIREsComputMolSci_002_0290_2012.pdf)

Perhaps important to mention is that the implemention in PyQint is not the most efficient one. There exist more efficient schemes. If you delve into the topic, you will certainly encounter them.
